enigma/src/test/java/org/quiltmc/enigma/input/z_tooltip/Constructors.java

```diff
@@ -0,0 +1,21 @@
+package org.quiltmc.enigma.input.z_tooltip;
+
+public class Constructors {
+	public Constructors(String outerArg) {
+		new Constructors("outer arg");
+
+		new Methods() {
+			// multiline declaration to test unindenting
+			@Deprecated
+			@Override
+			void abstraction() {
+				// tests #252
```

Comment:
not a comment on this PR but i'm going to make sure links like this have archived versions attached, i don't want any information lost if we ever have to move off github